src/api/parser/src/index.js

```diff
@@ -0,0 +1,111 @@
+const { Satellite } = require('@senecacdot/satellite');
+const { router } = require('bull-board');
+// May need in next revision
```

Comment:
What does this mean?

Reply:
This was supposed to have line 4 commented out as I wasn't sure if we'll need other routes aside from the bull-board